build_vol_cmake.sh

```diff
@@ -70,6 +65,8 @@ usage()
     echo "              library should be built with position independent"
     echo "              code option enabled."
     echo
+    echo "      -x      Enable building of the REST VOL examples."
```

Comment:
We should have building of the examples on by default instead of needing a build option for them. If they don't currently compile, we should fix that.